packages/shared/components/programs/program-details/program-details-description/program-details-description-section.js

```diff
@@ -104,13 +104,13 @@ class ProgramDetailsDescriptionSection extends PureComponent {
       <Fragment>
         <ProgramDetailsDescription
           onReinvestingClick={this.handleOnReinvestingClick}
-          programDescription={programDescription}
           isReinvestPending={ui.isReinvestPending}
           onFavoriteClick={this.handleOnFavoriteClick}
           isFavoritePending={ui.isFavoritePending}
           investmentData={composeInvestmentData(programDescription)}
           {...programDescription.personalProgramDetails}
           {...this.props}
+          programDescription={programDescription}
```

Comment:
Don't change the transfer of the props.
Prop "isReinvest" transfer from `{...programDescription.personalProgramDetails}`. Use him.